doc/release-notes/release-notes-0.16.1.md

```diff
@@ -110,6 +110,7 @@ weights.
 - #12442 `f3b8d85` devtools: Exclude patches from lint-whitespace (MarcoFalke)
 - #12988 `acdf433` Hold cs_main while calling UpdatedBlockTip() signal (skeees)
 - #12985 `0684cf9` Windows: Avoid launching as admin when NSIS installer ends. (JeremyRand)
+- #13004 `6a3b0d3` Print to console by default when not run with -daemon (practicalswift)
```

Comment:
Did you intend to update the historical release notes for `0.16.1`?

Reply:
Sorry for the late reply. I did intend that, because I figured the changes went into effect in 0.16.1 and that was the version that I was having problems with.